test-data/unit/fixtures/tuple.pyi

```diff
@@ -21,7 +21,8 @@ class function: pass
 class ellipsis: pass
 
 # We need int and slice for indexing tuples.
-class int: pass
+class int:
+    def __neg__(self) -> 'int': pass
```

Comment:
This was because of the one-line change in `checkexpr.py`: since we're deducing a type for index expressions now, a few unrelated tests about tuples and negative indices started failing.